test/e2e/from-import-ui.spec.js

```diff
@@ -335,9 +335,9 @@ describe('Using MetaMask with an existing account', function () {
     })
   })
 
-  describe('Connects to a Hardware wallet', () => {
-    it('choose Connect Hardware Wallet from the account menu', async () => {
-      const [connectAccount] = await driver.findElements(By.xpath(`//div[contains(text(), 'Connect Hardware Wallet')]`))
+  describe('Connects to an External  wallet', () => {
+    it('choose Connect External Wallet from the account menu', async () => {
+      const [connectAccount] = await driver.findElements(driver, By.xpath(`//div[contains(text(), 'Connect External Wallet')]`))
```

Comment:
```suggestion
      const [connectAccount] = await driver.findElements(By.xpath(`//div[contains(text(), 'Connect External Wallet')]`))
```
The `driver` param isn't needed anymore, now that this is being called from the `driver`.

Reply:
Thanks :), merged that by mistake, should be fixed now

Reply:
Great thanks!